src/fromager/commands/lint_requirements.py

```diff
@@ -27,10 +28,16 @@ def lint_requirements(input_files_path: list[click.Path]) -> None:
     flag = True
 
     for path in input_files_path:
-        parsed_lines = requirements_file.parse_requirements_file(str(path))
+        parsed_lines = requirements_file.parse_requirements_file(path)
+        unique_entries: set[str] = set()
         for line in parsed_lines:
             try:
-                Requirement(line)
+                requirement = Requirement(line)
+                if requirement.name in unique_entries:
+                    raise InvalidRequirement("Duplicate entry")
```

Comment:
For a duplicate it would be helpful to the user to show the line where the original entry appears.